target/linux/ramips/image/mt7620.mk

```diff
@@ -177,6 +177,21 @@ define Device/dlink_dwr-921-c1
 endef
 TARGET_DEVICES += dlink_dwr-921-c1
 
+define Device/dlink_dwr-921-c3
+  DTS := DWR-921-C1
```

Comment:
please use here

```
define Device/dlink_dwr-921-c3
  $(Device/dlink_dwr-921-c1)
  ...
```

and define only variables which differ from the dwr-921-c1 image build code.

Reply:
Done.